README.md

```diff
@@ -69,27 +154,32 @@ Now you can create a main file in the `source` and run your code with:
 dub
 ```
 
+Packages
+--------
+
+Putting `mir` in `/usr/include/dlang/dmd`, avoids the need for `-Isource`.
+So you can execute your code with `rdmd <your-file.d>`
+
 ### Arch Linux
 
 | Type   | Name         | Version  |
 |--------|--------------|----------|
 | Stable | `libmir`     | [![libmir](https://img.shields.io/aur/version/libmir.svg)](https://aur.archlinux.org/packages/libmir/) [![libmir](https://img.shields.io/aur/votes/libmir.svg)](https://aur.archlinux.org/packages/libmir/) |
 | Latest | `libmir-git` | [![libmir-git](https://img.shields.io/aur/version/libmir-git.svg)](https://aur.archlinux.org/packages/libmir-git/) |
 
-### On any platform with import paths
-
-Mir is a pure source code library, that means it can be easily distributed to
-any system. So you can just copy Mir's source to your system's dlang import path.
-For example on  Linux this is `/usr/include/dlang/dmd/mir`
-(or `/usr/include/dlang/dmd/mir` for ldc).
+Other tips
```

Comment:
We do not need this section because mir has very tiny name space for linking. Furthermore, please never refer from Mir related pages to articles about vibe.d (it is very buggy library and user may get bad experience )